dell-csi-helm-installer/csi-offline-bundle.md

```diff
@@ -78,13 +78,13 @@ For example, here is the output of a request to build an offline bundle for the
 *
 * Pulling and saving container images
 
-   dellemc/csi-isilon:v2.11.0
+   dellemc/csi-isilon:v2.12.0
    dellemc/csi-metadata-retriever:v1.6.0
-   dellemc/csipowermax-reverseproxy:v2.10.0
-   dellemc/csi-powermax:v2.11.0
```

Comment:
Hi @bjiang27 , How about changing the registry reference here?
![image](https://github.com/user-attachments/assets/e09374b7-44b3-480e-8afb-ef36244f1ec4)

Reply:
Thank you for your comment. The changes for quay.io image updates is in progress, and will be included in a separate pull request.